templates/download/xilinx/index.html

```diff
@@ -31,78 +31,293 @@ <h1>Install Ubuntu on Xilinx</h1>
+          </div>
+          <div class="col-6 u-align--center u-hide--small">
+            {{ image (
+              url="https://assets.ubuntu.com/v1/24e23fbe-zcu106-evaluation-board.png",
+              alt="Xilinx ZCU circuit board",
+              width="400",
+              height="273",
+              hi_def=True,
+              loading="auto"
+              ) | safe
+            }}
+          </div>
+        </div>
+
+        <div class="u-fixed-width">
+          <h4>First time installing Ubuntu on Xilinx?</h4>
+          <p>Please see the <a class="p-link--external" href="https://xilinx-wiki.atlassian.net/wiki/spaces/A/pages/2037317633/Getting+Started+with+Certified+Ubuntu+20.04+LTS+for+Xilinx+Devices?utm_source=canonical&utm_medium=referral&utm_campaign=ubuntu&utm_content=download&utm_term=desktop_image">Xilinx Getting Started Guide</a> for more information.</p>
+          <p><strong>Xilinx Ubuntu Software Development Kit</strong></p>
+          <p>Ubuntu Developers targeting Xilinx Zynq UltraScale+ MPSoC devices may also want to download the following files:</p>
+        </div>
+        <div class="u-fixed-width">
+          <table aria-label="Xilinx Ubuntu Software Development Kit table">
+            <thead>
+              <tr>
+                <th>Image name</th>
+                <th colspan="2">Description</th>
+              </tr>
+            </thead>
+            <tbody>
+              <tr>
+                <th><a href="https://people.canonical.com/~platform/images/xilinx/iot-zcu10x-classic-desktop-2004-x07-20210728-85-sysroot.tar.xz">iot-zcu10x-classic-desktop-2004-x07-20210728-85-sysroot.tar.xz</a></th>
+                <td colspan="2">Sysroot for cross-compilation</td>
+              </tr>
+              <tr>
+                <th><a href="https://people.canonical.com/~platform/images/xilinx/rootfs.ext4.xz">roofts.ext4.xz</a></th>
+                <td colspan="2">EXT4 formatted partition containing entire raw rootfs</td>
+              </tr>
+              <tr>
+                <th><a href="https://people.canonical.com/~platform/images/xilinx/rootfs.tar.gz">rootfs.tar.gz</a></th>
+                <td colspan="2">Raw Root filesystem</td>
+              </tr>
+              <tr>
+                <th><a href="https://people.canonical.com/~platform/images/xilinx/system-boot.tar.gz">system-boot.tar.gz</a></th>
+                <td colspan="2">FAT Partition Contents</td>
+              </tr>
+            </tbody>
+          </table>
+        </div>
+        <div class="u-fixed-width">
+          <p>If you have any questions about the platform or would like information about our certification program, please contact us.</p>
+          <p><a href="/download/contact-us" class="js-invoke-modal">Get in touch&nbsp;&rsaquo;</a></p>
+        </div>
+      </div>
+
+      <div tabindex="0" role="tabpanel" id="kira-kv260-vision-ai-tab" aria-labelledby="kira-kv260-vision-ai" hidden="true">
+        <div class="row u-vertically-center">
```

Comment:
Same as previous comment:
```suggestion
        <div class="row u-vertically-center u-no-padding">
```